lib/commands/web.js

```diff
@@ -231,89 +263,28 @@ helpers.executeAtomAsync = async function executeAtomAsync (atom, args, response
   return await this.waitForAtom(promise);
 };
 
-helpers.getAtomsElement = function getAtomsElement (wdId) {
-  let atomsId;
-  try {
-    atomsId = this.webElementIds[parseInt(wdId, 10) - ELEMENT_OFFSET];
-  } catch (e) {
-    return null;
+helpers.getAtomsElement = function getAtomsElement (elOrId) {
+  const elId = util.unwrapElement(elOrId);
+  if (!this.webElementsCache.has(elId)) {
+    throw new errors.StaleElementReferenceError();
   }
-  if (_.isUndefined(atomsId)) {
-    return null;
-  }
-  return {ELEMENT: atomsId};
-};
-
-helpers.useAtomsElement = function useAtomsElement (el) {
-  if (parseInt(el, 10) < ELEMENT_OFFSET) {
-    log.debug(`Element with id '${el}' passed in for use with ` +
-      `atoms, but it's out of our internal scope. Adding ${ELEMENT_OFFSET}.`);
-    el = (parseInt(el, 10) + ELEMENT_OFFSET).toString();
-  }
-  let atomsElement = this.getAtomsElement(el);
-  if (atomsElement === null) {
-    throw new errors.UnknownError(`Error converting element ID for using in WD atoms: '${el}'`);
-  }
-  return atomsElement;
+  return {ELEMENT: this.webElementsCache.get(elId)};
```

Comment:
while we're changing things here, do we want to upgrade this to use the w3c element key?

Reply:
this method is used to fetch element identifier for usage with atoms. I don't think atoms know anything about w3c though